source/projects/overview.txt

```diff
@@ -0,0 +1,50 @@
+.. _rm-project-overview:
+
+=======================
+User Interface Overview
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+When you open a project in Relational Migrator, the project user interface displays.
+You can manage many aspects of your Relational Migrator project from this interface:
+
+.. figure:: /img/projects/project-overview.png
+   :alt: Project Overview Screen
+   :lightbox:
+
+UI Elements
+-----------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - UI Element
+     - Description
+
```

Comment:
(suggestion)
For each of the descriptions, we list at least two actions that can be taken. What about turning each into a bulleted list?

Reply:
I think the way the page is written for "Description" makes this look a little odd when I staged it. It is more scan-able, but given that none of the descriptions are more than 3 sentences I will keep this as is.